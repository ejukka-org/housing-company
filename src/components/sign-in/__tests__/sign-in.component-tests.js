import React from "react";
import ReactDOM from "react-dom";
import SignIn from "../sign-in.component";
import { act } from "react-dom/test-utils";
import { Provider } from "react-redux";
import reduxStore from "../../../redux/store";
import pretty from "pretty";

let container;

beforeEach(() => {
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  document.body.removeChild(container);
  container = null;
});

test("Renders SignIn component", () => {
  const store = reduxStore.store;
  act(() => {
    ReactDOM.render(
        <Provider store={store}>
          {" "}
          <SignIn />{" "}
        </Provider>,
        container
    );
  });
  expect(pretty(container.innerHTML)).toMatchSnapshot();
});
