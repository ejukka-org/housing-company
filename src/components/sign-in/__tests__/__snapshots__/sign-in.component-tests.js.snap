// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders SignIn component 1`] = `" <div class=\\"container\\"><br><button type=\\"button\\" class=\\"btn btn-primary\\">Kirjaudu Googlen tunnuksilla</button></div>"`;
